File: NewProject/wwwroot/js/Class/Spjjz.js
//商品禁忌症
export default class Spjjz {
    constructor() {
        //禁忌症ID
        this.jjzbh = "";
        //禁忌症名称
        this.jjzmc = "";
    }
    addData(obj) {
        this.jjzbh = obj.jjzbh;
        this.jjzmc = obj.jjzmc;
    }
}

File: NewProject/Models/SPJJZ.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;


namespace NewProject.Models
{
    //禁忌症
    public class SPJJZ : Table
    {
        [Key]
        public int GUID { get; set; }
        [Display(Name = "禁忌症ID")]
        public string JJZBH { get; set; }
        [Display(Name = "产品ID")]
        public int SPBH { get; set; }
        [Display(Name = "禁忌症名称")]
        public string JJZMC { get; set; }
    }
}
